np.from_dlpack was not added until numpy 1.23
Patch based off https://github.com/google/jax/pull/12900 and added by Simon Branford (University of Birmingham)
diff --git a/tests/array_interoperability_test.py b/tests/array_interoperability_test.py
index cc2f3a6c08d..ac04ee637c8 100644
--- a/tests/array_interoperability_test.py
+++ b/tests/array_interoperability_test.py
@@ -200,7 +200,7 @@ def testNumpyToJax(self, shape, dtype):
      "shape": shape, "dtype": dtype}
      for shape in all_shapes
      for dtype in torch_dtypes))
-  @unittest.skipIf(numpy_version < (1, 22, 0), "Requires numpy 1.22 or newer")
+  @unittest.skipIf(numpy_version < (1, 23, 0), "Requires numpy 1.23 or newer")
   @jtu.skip_on_devices("gpu")
   def testJaxToNumpy(self, shape, dtype):
     rng = jtu.rand_default(self.rng())
